### PR TITLE
Made v8 heap size ("old stack size") configurable and BGJSGLView fix

### PR DESCRIPTION
**Pull request by unverbraucht**
Heap size is set per default to 1/3 of app heap size.
Memory optimizations: dispose touch event objects eagerly.
Bug fix: run the whole rendering step in one v8::Locker.

### CODE REVIEW

File: src/main/java/ag/boersego/bgjs/BGJSGLView.kt [hunk 1]
```diff
@@ -106,16 +114,18 @@ open class BGJSGLView(engine: V8Engine, val textureView: V8TextureView) : JNIV8O
             callbacksForThisRedraw.addAll(callbacksRedraw)
             queuedAnimationRequests.forEach { callbacksForThisRedraw.add(it.cb) }
             queuedAnimationRequests.clear()
-        }
 
-        didDraw = !callbacksForThisRedraw.isEmpty()
+            didDraw = !callbacksForThisRedraw.isEmpty()
+
+            // TODO: what are on.redraw?
```
Comment (Tharit) [inline src/main/java/ag/boersego/bgjs/BGJSGLView.kt]: See explanation provided via messenger; should definitely not be called here.
Instead we should have something like "fun requestRedraw" that would trigger all the callbacks.
This method would be unused for now, but could be called by the app, or by the library in the future should the texture view be reset for any reason other than resizing..

File: src/main/java/ag/boersego/bgjs/V8TextureView.java [hunk 2]
```diff
@@ -324,6 +324,12 @@ private void sendTouchEvent(String type) {
         touchEventObj.setV8Field("touches", touches);
 
         mBGJSGLView.onEvent(touchEventObj);
+
+        for (final JNIV8GenericObject touch : touchObjs) {
+        	touch.dispose();
+		}
```
Comment (Tharit) [inline src/main/java/ag/boersego/bgjs/V8TextureView.java]: looks like wrong indentation ;-)

File: src/main/java/ag/boersego/bgjs/BGJSGLView.kt [hunk 3]
```diff
@@ -106,16 +114,18 @@ open class BGJSGLView(engine: V8Engine, val textureView: V8TextureView) : JNIV8O
             callbacksForThisRedraw.addAll(callbacksRedraw)
```
Comment (Tharit) [inline src/main/java/ag/boersego/bgjs/BGJSGLView.kt]: Should not be used here; see explanation provided in  messenger;
we should have something like "fun requestRedraw" that we could call to tell the javascript layer to reqeust a new animation frame and render should we ever need to reinitialize the opengl context for some reason other than resize.

File: src/main/java/ag/boersego/bgjs/V8TextureView.java (hunk 2 above)
Comment (unverbraucht) [reply src/main/java/ag/boersego/bgjs/V8TextureView.java]: There's a lot of old tab indentation. It looks alright in the IDE with the correct tab size. Maybe we should do a separate tabs-to-spaces sweep commit.

**Pull request merged**